sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2840,6 +2938,14 @@ public partial class GremlinGraphResourceInfo
         public Azure.ResourceManager.CosmosDB.Models.CosmosDBContainerPartitionKey PartitionKey { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.ResourceManager.CosmosDB.Models.CosmosDBUniqueKey> UniqueKeys { get { throw null; } }
     }
+    public partial class KeyWrapMetadata
```

Comment:
Shall we rename this class to `CosmosDBKeyWrapMetadata`?

Reply:
@kr-santosh please take a look at this.

Reply:
This class is not properly renamed, could we get this renamed?